src/string_util.cc

```diff
@@ -169,4 +169,93 @@ void ReplaceAll(std::string* str, const std::string& from,
   }
 }
 
+#ifdef BENCHMARK_STL_ANDROID_GNUSTL
```

Comment:
can you add tests for these please? 

you'll have to create a test/string_util_gtest.cc file, and add it to test/CMakeLists.txt. bazel will pick it up automatically.

Reply:
Added tests and replaces `log2(n)` with `1.44269504088896340736 * log(n)` as Android GNUSTL lacks that too.